README.md

```diff
@@ -21,6 +21,51 @@ _**Note**: below listed features are still under design and review phase and hen
 * New range functions
 * Freeze rows/columns.
 
+## Upcoming Excel 1.8 Event Release Features
+
+Below is the early thought of Event included in Excel JavaScript APIs. Further details about event argument design and sample code will be posted later. 
```

Comment:
Revise as "This section describes our early thoughts about events in the Excel JavaScript API. We'll add more details about event argument design and code examples in the future."